packages/mask/src/components/InjectedComponents/PostReplacer.tsx

```diff
@@ -1,9 +1,7 @@
-import { useActivatedPluginsSNSAdaptor } from '@masknet/plugin-infra'
-import { isTypedMessageTuple, isWellKnownTypedMessages } from '@masknet/shared-base'
 import { makeStyles } from '@masknet/theme'
-import { useEffect, useMemo } from 'react'
+import { useEffect } from 'react'
 import { usePostInfoDetails } from '../DataSource/usePostInfo'
-import { PayloadReplacerTransformer } from './PayloadReplacer'
+import './PayloadReplacer'
```

Comment:
?

Reply:
This PR is not ready for review yet, it's still WIP.